Src/Newtonsoft.Json/Converters/JavaScriptDateTimeConverter.cs

```diff
@@ -30,7 +30,7 @@
 namespace Newtonsoft.Json.Converters
 {
     /// <summary>
-    /// Converts a <see cref="DateTime"/> to and from a JavaScript date constructor (e.g. new Date(52231943)).
+    /// Converts a <see cref="DateTime"/> to and from a JavaScript <c>Date</c> constructor (e.g. <c>"new Date(52231943)"</c>).
```

Comment:
There shouldn't be quotes around the new Date. It isn't written to JSON as a string